### DISCUSSION TITLE
Can I run EMu on a Mac?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#419] [FAQs] [answered]
Can I run EMu on a Mac?

**Comment by reguzik** [comment, marked as answer]
Yes, but not natively.

1) The recommended way to run EMu from a Mac is to use one of the Museum's virtual machines (VMs) for a session. If you have never used a remote desktop, don't worry, contact the [helpdesk@fieldmuseum.org](mailto:helpdesk@fieldmuseum.org) and they will get you set up. [Instructions are available](https://docs.google.com/document/d/1CAtX0vOZmNYc_XsNXNXKJUyhGV4x9ztcftozxuAjB2A) 

2) If you have used VMWare Fusion, Parallels or Virtual box in the past, these options are no longer supported by the IT department. Please contact the helpdesk for help transitioning to using the Museum's VMs. 

- Please contact [helpdesk@fieldmuseum.org](mailto:helpdesk@fieldmuseum.org) before setting any of these up.

You will also need an up to date Mac to run everything. These are the Jan 2016 recommended Mac specs:

- Mac Pro or Mac Book Pro (if you have a Mac Book Pro, will need a Thunderbolt to Ethernet Adapter)
- 500 GB HDD
- Between 8-16 GB RAM (accommodates for both OS's functioning on a high level, shouldn't slow down the Mac side)
- Core i5 Processor
- Any OS X beyond 10.8.5 should be capable of handling VMWare Fusion 8. This encompasses Mavericks, Yosemite, El Capitan.


NOTE: For any IT purchases made outside of SHI you are responsible for arranging tax exemption. [More information](http://intranet.fieldmuseum.org/finance/node/146) is available on how to do this from finance.

**Comment by peteherbst** [comment]
If you have an Apple Silicon (M1, etc.) Mac, verify that your virtualization software can run on your Mac.

These pieces of software were originally designed to run on Intel processors, so they are not yet available for Apple Silicon (as of April 2022).
